palm/plugins/core/templates/containerize/entrypoint.sh.txt

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash -x
+# Entrypoint script for docker
+{% if package_manager == 'poetry' %}
+pip3 install poetry
+poetry config virtualenvs.create false
+poetry install
+{% endif %}
+{% if package_manager == 'pip3' %}
+pip3 install -r requirements.txt
+pip3 install pytest
```

Comment:
do we want to leave this up to them in the requirements.txt? they might want to do a cascade and include it in a dev-requirements.txt or similar at some point. 

future thoughts really.

Reply:
Yeeeeaahhhh... I could definitely be convinced to change this.

I originally had some logic here to pass an arg to this script and do something like this:

```
if [ -z "$1" ] ; then 
    # Not in deploy mode
    pip install -r requirements.txt
    pip install pytest
else 
    # Deploy mode
    pip install -r requirements.txt
fi
```

but it felt super opinionated, so I decided to simplify down to the bare minimum. Since it's common to use pytest for testing, and palm is a local-dev tool I wanted to make sure tests would run.